NDTensors/test/NDTensorsTestUtils/device_list.jl

```diff
@@ -18,13 +19,21 @@ if "metal" in ARGS || "all" in ARGS
 end
 if "cutensor" in ARGS || "all" in ARGS
   Pkg.add("cuTENSOR")
+  Pkg.add("CUDA")
```

Comment:
```suggestion
  Pkg.add("CUDA")
  Pkg.add("cuTENSOR")
```
Alphabetical ordering.

Reply:
I wasn't sure if this was going to work. I have previously run into trouble in `Pkg.test` when I tried to add things to the manifest during a test. So far this seems to work though it may be causing the issues we see on Jenkins. Regardless I have reorganized and fixed the `using` calls

Reply:
These are working now from what I can tell, I am closing this comment